packages/starlight/utils/git.ts

```diff
@@ -39,3 +40,76 @@ export function listGitTrackedFiles(directory: string): string[] {
 	const output = result.stdout.trim();
 	return output.split('\n').map((file) => join(directory, file));
 }
+
+function getRepoRoot(directory: string): string {
```

Comment:
If I am correct, with this new refactor, `listGitTrackedFiles()` (just above but outside of review range) is no longer used? If correct, we should remove it entirely.